tests/test_node_differ.py

```diff
@@ -132,10 +143,19 @@ def test_change_text_count_english_punctuations():
                         "Whitespace":{},"Punctuation":{'change':1},
                         'Paragraph':{'change':1}
                         }
-    get_text_structure = nd.parse_change_text(prev_text,curr_text,'Text')
+    get_text_structure = nd.parse_change_text('Text', prev_text,curr_text)
     assert expected_changes == get_text_structure
 
 
+    
+def test_text_change():
+    curr_wikitext = prev_wikitext.replace('Aigen was born in Olomouc on 8 October 1685, the son of a goldsmith.',
+                                          'Aigen-Abe was born in Olomouc on 9 October 1685, the daughter of a goldsmith.',
+                                          1)
+    expected_changes = {'Paragraph':{'change':1}, 'Sentence':{'change':1}, 'Word':{'change':3},'Punctuation':{'change':1},'Whitespace':{},'Punctuation':{'change':1}}
```

Comment:
Fine for now but for a future meeting, let's talk through whether a hyphen should split a word into two or not (which I think is how it's currently parsed based on the expected changes)